### ISSUE TITLE
Spike: Assess confidence in ability for Lockwise to capture new and update existing accounts created within Fenix

### ISSUE THREAD

**Issue by changecourse** [mozilla-lockwise/lockwise-android#1108]
We need to ensure that we have confidence about capture for new and updating existing credentials within Fenix via Lockwise

Specifically, that a Lockwise modal will show up in **each and every instance** where someone creates a new account (in parity, or better than Fenix) as well as instances where an account needs to be updated (new password)

This ensures that the work the Fenix team is doing enables Lockwise to handle login management, rather than a reduced experience with our app.

**Comment by eliserichards** [comment]
Is this meant to be an engineering task? Not sure what the definition of "done" here would be 🤔

**Comment by sandysage** [comment]
@eliserichards I think this is intended to be a follow up to a discussion had with @jhugman about leveraging Firefox's fill/capture heuristics in toolkit in Lockwise. I think the idea here is to determine if that really is feasible, and if so, scope out the work to augment our fill/capture abilities with the Firefox heuristics.

**Comment by changecourse** [comment]
@eliserichards yes, likely we'll need to create a spreadsheet of top sites, with use cases for both creating new and updating existing accounts, and compare the reliability of capture between Fenix and Lockwise.

**Comment by eliserichards** [comment]
Spreadsheet: https://docs.google.com/spreadsheets/d/1cOHLE9-GxppEQ0jLURa945WBnryMDA7a6yBn3ogMe8c/edit#gid=0

**Comment by eliserichards** [comment]
Autofill in fenix is available in nightly currently (hoping to land in the next release).

**Comment by eliserichards** [comment]
Out of 50 of the top trending current **apps**, only four do not work with the Lockwise autofill service, three of which are phone number logins which is a data type that is not currently supported by Sync/FxA.

Out of 50 of the top trending **websites** (US known sites), only five are not working with Lockwise autofill on the current version of the Fenix browser (90% success rate). This number is dramatically different than the autofill performance on Fennec, which, while still impressive, functions on only 34 out of the same 50 websites (68% success rate).
